README.md

```diff
@@ -73,6 +73,7 @@ Additional resources on MCP.
 - **[Awesome MCP Servers by punkpeye](https://github.com/punkpeye/awesome-mcp-servers)** - A curated list of MCP servers by **[Frank Fiegel](https://github.com/punkpeye)**
 - **[Awesome MCP Servers by wong2](https://github.com/wong2/awesome-mcp-servers)** - A curated list of MCP servers by **[wong2](https://github.com/wong2)**
 - **[Awesome MCP Servers by appcypher](https://github.com/appcypher/awesome-mcp-servers)** - A curated list of MCP servers by **[Stephen Akinyemi](https://github.com/appcypher)**
+- **[Discord Server](https://glama.ai/mcp/discord)** – A community discord server dedicated to MCP by **[Frank Fiegel](https://github.com/punkpeye)**
```

Comment:
Can you please change this into a direct Discord link? I think it's a little weird to go through the intermediary here.

Reply:
Okay, makes sense. Thanks for explaining.

Reply:
You can make the link never expire though?

https://github.com/user-attachments/assets/f5fc18f9-a535-4d65-a249-a8112abb6b55